java/light-4j/pom.xml

```diff
@@ -0,0 +1,428 @@
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>benchmark.light-4j</groupId>
+    <artifactId>benchmark</artifactId>
+    <packaging>jar</packaging>
+    <name>benchmark</name>
+    <version>0.0.1</version>
+
+    <properties>
+        <java.version>1.8</java.version>
+        
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <project.main.class>com.networknt.server.Server</project.main.class>
+        <version.light-4j>1.6.23</version.light-4j>
+        <version.jackson>2.10.0</version.jackson>
+        <version.slf4j>1.7.25</version.slf4j>
+        <version.jose4j>0.6.3</version.jose4j>
+        <version.logback>1.2.3</version.logback>
+        <version.junit>4.12</version.junit>
+        <version.undertow>2.0.29.Final</version.undertow>
+        <version.json-schema-validator>1.0.29</version.json-schema-validator>
+        <version.fastscanner>2.18.1</version.fastscanner>
+        
+        
+        
+        
+        
+        <versions.maven-version>2.4</versions.maven-version>
```

Comment:
Is this not provided globally ? I mean by https://github.com/the-benchmarker/web-frameworks/blob/master/java/Dockerfile (without any specific image in `config.yaml` mave **3.6** is used)

Reply:
This property name is a bit misleading; I believe the version is for the [Versions Maven Plugin](https://www.mojohaus.org/versions-maven-plugin/), not the version of Maven itself. I don't think it is actually necessary here so I've removed both the property and the plugin itself (in b81fb2691f80630445b1add4f978e939c0180966).